README.md

```diff
@@ -504,13 +484,7 @@ This project follows the [all-contributors](https://github.com/kentcdodds/all-co
 [validators-docs]:      http://uilibrary-docs.terminus.ninja/release/injectables/TsValidatorsService.html
 
 <!-- Versioned Docs & Demos -->
-[docs-v8]: http://uilibrary-docs.terminus.ninja/8.x.x/
-[docs-v9]: http://uilibrary-docs.terminus.ninja/9.x.x/
-[docs-v10]: http://uilibrary-docs.terminus.ninja/10.x.x/
 [docs-latest]: http://uilibrary-docs.terminus.ninja/release/
-[demo-v8]: https://getterminus.github.io/ui-demos-8.x.x/
-[demo-v9]: https://getterminus.github.io/ui-demos-9.x.x/
-[demo-v10]: https://getterminus.github.io/ui-demos-10.x.x/
 [demo-latest]: https://getterminus.github.io/ui-demos-release/
```

Comment:
didn't realize we had all those version docs listed here...

Reply:
Yeah, I think I originally left the links here thinking we'd likely need them again. But thankfully we haven't :)